lib/fetch-package-metadata.js

```diff
@@ -56,13 +89,15 @@ function fetchPackageMetadata (spec, where, opts, done) {
   if (!npmConfig) {
     npmConfig = require('./config/figgy-config.js')
```

Comment:
If this (figgy-pudding-ish) `npmConfig` is only used in this file for pacote options we could strip figgy-pudding completely from this file and add defaults/fetch options on a plain object.